### PR TITLE
Migrate /support, refactor component

### PR DESCRIPTION
**Pull request by sandylcruz**
* Migrate https://goteleport.com/support/
* `/support` preview here: https://teleport-docs-next-git-sandy-modify-component-gravitational.vercel.app/support/
* Change name of  `ListAndImage` to `ImageAndContent`. Refactor it to be able to take in more input than a list. Allow for positioning of image to the right or left
* Use `ImageAndContent` in `/security` page preview here: https://teleport-docs-next-git-sandy-modify-component-gravitational.vercel.app/security/
* Add storybook entry for `ImageAndContent`

## Desktop
Screen Shot 2021-11-19 at 4 54 24 PM

## Smaller screen
Screen Shot 2021-11-19 at 4 57 49 PM

## Mobile device
Screen Shot 2021-11-19 at 4 56 17 PM

### CODE REVIEW

File: components/ImageAndContent/ImageAndContent.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                {subtitle}
+              </Box>
+              <Flex color="darkest" mb={2}>
+                {subheader}
+              </Flex>
+
+              {/* Lower Content */}
+              <Flex
+                flexDirection="row"
+                mr={imageToRightAndNotCentered ? "90px" : 3}
```
Comment (C-STYR) [inline components/ImageAndContent/ImageAndContent.tsx:78]: Any numerical values that are greater than the highest index position in the space array in `theme.ts` don't need to be strings with units, just FYI!

```suggestion
                mr={imageToRightAndNotCentered ? 90 : 3}
```

File: pages/support/index.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+title: Teleport Support and Education Center
+subtitle: Help & Support
+description: "Educational Material, Community Questions & Answers, and Contact Information."
+headerBehaviour: floating
+headerMode: full
+hideWave: true
+---
+
+import styled from "styled-components";
+import css from "@styled-system/css";
```
Comment (C-STYR) [inline pages/support/index.mdx:11]: I think these can be removed...?

File: pages/support/index.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  <ImageAndContent
+    imagePosition="left"
+    listHeading="Documentation and resources that can provide you with an immediate answer"
+    subheader="Teleport Documentation"
+    subtitle="Documentation and resources that can provide you with an immediate answer"
+    title="Education & Training"
+    content={
+      <ul>
+        <li>
+          <Link href="/docs/getting-started/" color="light-purple">
```
Comment (C-STYR) [inline pages/support/index.mdx:45]: For all links, please remove `color` prop and insert `scheme="site"`

**Review by C-STYR — APPROVED** [review]
Nice work.  Approved with comments...

File: components/ImageAndContent/ImageAndContent.tsx (hunk 1 above)
Comment (sandylcruz) [reply components/ImageAndContent/ImageAndContent.tsx:78]: did not know that! updated

File: pages/support/index.mdx (hunk 2 above)
Comment (sandylcruz) [reply pages/support/index.mdx:11]: yep, removed

File: components/ImageAndContent/ImageAndContent.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  content,
+  imagePosition = "right",
+  imageToRightAndNotCentered = false,
+  subheader,
+  subtitle,
+  title,
+}: ImageAndContentProps) => {
+  return (
+    <Section
+      bg="grayGradient"
```
[the remark this answers is not in the document]
Comment (sandylcruz) [reply components/ImageAndContent/ImageAndContent.tsx:27]: that makes sense, updated it so you can alter BGColor

**Pull request merged**